package.json

```diff
@@ -3,25 +3,28 @@
   "version": "2.1.1",
   "private": false,
   "type": "module",
+  "types": "./index.d.ts",
   "main": "./cjs/index.js",
+  "module": "./esm/index.js",
   "exports": {
     "./package.json": "./package.json",
     ".": {
-      "types": "./index.d.ts",
+      "types": {
+        "require": "./cjs/index.d.js",
```

Comment:
not .ts ?

Reply:
Oops. My bad.

Anyways, thanks for releasing v2.1.2.